libs/model-assessment/src/lib/ModelAssessmentDashboard/Context/buildModelAssessmentContext.ts

```diff
@@ -60,6 +60,7 @@ export function buildInitialModelAssessmentContext(
     featureMetaData: props.dataset.feature_metadata,
     localExplanations,
     metadata: modelMetadata,
+    objectDetectionLabels: props.dataset.objectDetectionLabels,
```

Comment:
This should be object_detection_labels, not objectDetectionLabels.  It looks inconsistent with all of the other fields.  We should add it to whitelist for linting like all of the other fields here, such as feature_metadata.

Reply:
Yep, will merge https://github.com/microsoft/responsible-ai-toolbox/pull/2360 first which adds this support.